specification/manufacturingdatasolution/mds-service/data-plane/Microsoft.Industries.ManufacturingDataSolution/stable/2025-03-01/openapi.json

```diff
@@ -0,0 +1,3593 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/QueryModels.QueryRequestRelatedEntitiesData"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/QueryModels.PagedRelatedEntityData"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Azure.Core.Foundations.ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "QueryApis_QueryRelatedEntitiesDataHistory": {
+            "$ref": "./examples/QueryApis_QueryRelatedEntitiesDataHistory.json"
+          }
+        }
+      }
+    },
+    "/query:relatedNodes": {
+      "post": {
+        "operationId": "QueryApis_QueryRelatedNodes",
+        "description": "Query the related nodes.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/QueryModels.QueryRequestRelatedNodes"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/QueryModels.PagedRelatedNodeResult"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Azure.Core.Foundations.ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "QueryApis_QueryRelatedNodes": {
+            "$ref": "./examples/QueryApis_QueryRelatedNodes.json"
+          },
+          "QueryApis_QueryRelatedNodes_MinimumSet": {
+            "$ref": "./examples/QueryApis_QueryRelatedNodes_MinimumSet_Gen.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AdminModels.CleanupJobStatusResponse": {
+      "type": "object",
+      "description": "Represents the response status of a cleanup job.",
```

Comment:
Here and for all the schemas, please drop "Represents" from the description. It is unnecessary and just adds bulk with no value.